Security/src/EOMT.ps1

```diff
@@ -790,7 +790,8 @@ try {
         #Normal run
         $IsVulnerable = Get-ServerVulnStatus
         if ($IsVulnerable) {
-            $Message = "$env:computername is vulnerable: applying mitigation"
+            $Message = "$env:computername is vulnerable: needs to apply mitigation"
+            $Message + "`n" + "Please use Microsoft Update to install the latest Security Update For Exchange Server (KB5000871)"
```

Comment:
This should be $Message += (not +). Please also combine the "`n" and the message into a single string.